### PR TITLE
Broke into components.

### PR DESCRIPTION
**Pull request by sumitdhungana14**

### CODE REVIEW

File: src/pages/Signup.svelte [hunk 1]
```diff
@@ -51,79 +54,57 @@
...
-                    <path
-                      fill-rule="evenodd"
-                      d="M5 9V7a5 5 0 0110 0v2a2 2 0 012 2v5a2 2 0 01-2 2H5a2 2 0 01-2-2v-5a2 2 0 012-2zm8-2v2H7V7a3 3 0 016 0z"
-                      clip-rule="evenodd" />
-                  </svg>
-                </span>
-                create my account
-              </button>
-            </div>
+            <Button>Create my account</Button>
```
Comment (isNan909) [inline src/pages/Signup.svelte:100]: @sumitdhungana14 Let's use this text as a prop to our button component as well.

File: src/components/Button.svelte [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          viewBox="0 0 20 20"
+          fill="currentColor"
+          aria-hidden="true">
+          <path
+            fill-rule="evenodd"
+            d="M5 9V7a5 5 0 0110 0v2a2 2 0 012 2v5a2 2 0 01-2 2H5a2 2 0 01-2-2v-5a2 2 0 012-2zm8-2v2H7V7a3 3 0 016 0z"
+            clip-rule="evenodd" />
+        </svg>
+      </span>
+      <slot>Submit</slot>
```
Comment (isNan909) [inline src/components/Button.svelte:23]: Lets replace this part with the prop from button component for the inner text.

**Review by isNan909 — CHANGES_REQUESTED** [review]
Some minor tweak and we are good to go. Thanks @sumitdhungana14

File: src/pages/Signup.svelte (hunk 1 above)
Comment (sumitdhungana14) [reply src/pages/Signup.svelte:100]: slot would be a better use case here IMO.

File: src/components/Button.svelte (hunk 2 above)
Comment (isNan909) [reply src/components/Button.svelte:23]: Ok sloths seems ok.

**Review by isNan909 — APPROVED** [review]
Thankyou for your effort.

**Pull request closed without merging**